src/metabase/lib/normalize.cljc

```diff
@@ -35,13 +35,15 @@
 
 (def default-map-value-fns
   "Default normalization functions keys when doing map normalization."
-  {:base-type   keyword
-   :type        keyword
+  {:base-type      keyword
+   :effective-type keyword
```

Comment:
Missing pMBQL normalization logic for `:effective-type` and `:semantic-type`. Fixed